tests/tests/test-ethers.ts

```diff
@@ -2,18 +2,30 @@ import { createAndFinalizeBlock, describeWithMoonbeam } from "./util";
 import { HttpProvider } from "web3-core";
 import { expect } from "chai";
 import { ethers } from "ethers";
-import { TEST_CONTRACT_ABI_ETHERS, TEST_CONTRACT_BYTECODE } from "./constants";
+import {
+  TEST_CONTRACT_ABI_ETHERS,
+  TEST_CONTRACT_BYTECODE,
+  GENESIS_ACCOUNT_PRIVATE_KEY,
+} from "./constants";
 
 describeWithMoonbeam("Moonbeam RPC (Ethers.js)", `simple-specs.json`, (context) => {
-  const GENESIS_ACCOUNT_PRIVATE_KEY =
-    "0x99B3C12287537E38C90A9219D4CB074A89A16E9CDB20BF85728EBD97C343E342";
-
-  it("get network id", async function () {
-    expect((await context.ethers.getNetwork()).chainId).to.equal(1281);
+  let provider;
```

Comment:
The context contains a ether provider now, there shouldn't be the need to create a new one here